dotcom-rendering/src/components/Caption.tsx

```diff
@@ -33,7 +27,7 @@ const captionStyle = css`
 	${textSans.xsmall()};
 	line-height: 135%;
 	padding-top: 6px;
-	overflow-wrap: break-all;
+	overflow-wrap: break-word;
```

Comment:
Far as I can tell from [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) `break-all` is not a valid value. I think we meant break-word.